api/pkg/client/mlflow/mlflow.go

```diff
@@ -0,0 +1,188 @@
+package mlflow
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+type mlflowClient struct {
+	Client *http.Client
+	Config Config
+}
+
+type Config struct {
+	TrackingURL string
+}
+
+type Mlflow interface {
+	SearchRunForExperiment(idExperiment string) (SearchRunsResponse, error)
+	SearchRunData(idRun string) (SearchRunResponse, error)
+	DeleteExperiment(idExperiment string) error
+	DeleteRun(idRun string) error
+}
+
+type DeleteExperimentRequest struct {
+	ExperimentId string `json:"experiment_id" required:"true"`
+}
+
+type DeleteRunRequest struct {
+	RunId string `json:"run_id" required:"true"`
+}
+
+type DeleteExperimentErrorResponse struct {
+	ErrorCode string `json:"error_code"`
+	Message   string `json:"message"`
+}
+
+type SearchRunRequest struct {
+	ExperimentId []string `json:"experiment_ids" required:"true"`
+}
+
+type TagRun struct {
```

Comment:
Nit: I think `RunTag`, `RunInfo` and `RunData` would be more logical (prefix with `Run` instead of suffix) to associate with the "run" entity.